### PR TITLE
Moved addons into `src/addons` and added VSCode settings and app launcher scripts

### PR DESCRIPTION
**Pull request by jack-yao91**
* Moved addons into `src/addons` so we can use blender script directories to source the code directly into blender like discussed [#29](https://github.com/poly-hammer/BlenderTools/issues/29)
* Had to update the tests and release scripts to accomodate this change
* Added `.vscode` folder for vscode users. These also include the launch tasks and debug configurations to remote debug blender and unreal.
* Updated the developer docs to include this

### CODE REVIEW

File: src/addons/send2ue/core/utilities.py
Comment (JoshQuake) [inline src/addons/send2ue/core/utilities.py:1]: will removing the chmod line potentially cause file permission issues?

**Review by JoshQuake — APPROVED** [review]
b-e-a-utiful!

File: src/addons/send2ue/core/utilities.py
Comment (jack-yao91) [reply src/addons/send2ue/core/utilities.py:1]: It shouldnt, these chmod calls were causing permissions issues. They shouldnt need 777 to delete the files so I removed them.

**Comment by jack-yao91** [comment]
New updates to dev docs. You can now remote debug blender and unreal and the tests. 
https://poly-hammer.github.io/BlenderTools/contributing/development/#vscode

Also you can remote debug the containers too. I made the containers public if people need to debug containers locally
https://github.com/orgs/poly-hammer/packages

**Pull request merged**